README.md

```diff
@@ -0,0 +1,33 @@
+# Open Trip Planner Client Library [![Java with Gradle](https://github.com/CUTR-at-USF/opentripplanner-client-library/actions/workflows/gradle.yml/badge.svg)](https://github.com/CUTR-at-USF/opentripplanner-client-library/actions/workflows/gradle.yml)
+
+A full-featured kotlin multiplatform library designed to minimize developer friction in creating complex API calls to the Open Trip Planner v2 web service.
+
+## Prerequisites
+1) Install Android Studio (preferably version `4.1.3`).
+2) Before you import the project, install Kotlin MultiPlatform Mobile plugin. You can install it by navigating through Settings (Preferences if on MacOS) -> Plugins -> Kotlin Multplatform Mobile -> Install
+
+## Build
+Import the project and click on the green play button on top.
+
+## Testing
+1) If you're testing on a Physical device, make sure to change the host to `localhost` or `127.0.0.1` in the API constructors and also make note of the port number.
+2) After installation, pay attention to the logs. The json responses will be logged in the `debug` with the tag `MainActivity`.
+
+## License
```

Comment:
It looks like if I run `./gradlew :shared:build` it generates AAR files (one for debug and one for release) in `shared\build\outputs\aar`.

Can you add a "Release" section too and document this? We'll need it when trying to publish a release as an artifacts.

As an aside, to use this in a Java desktop app, we may need another section equivalent to the "iOS" and "Android" parts in the library module. I would expect the library to generate a JAR for use on desktop Java, as desktop Java won't understand an AAR (AFAIK). But let's tackle this in another issue/PR - could you open an issue for it?